src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: bold;
+}
+
+.form {
+  margin-top: auto;
+}
+
+.form-input {
+  display: block;
+  margin: o;
+  width: 100%;
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+  font-size: 16px; /* розмір шрифтк */
+  color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  box-shadow: rgba(61, 78, 97, 0.1);
```

Comment:
The `box-shadow` property is missing horizontal and vertical offsets, and the blur radius. It should be something like `box-shadow: 0 1px 4px rgba(61, 78, 97, 0.1);`.